src/components/navigation/navbar/desktop/admin/administration/ManageAccountsMenuBody.vue

```diff
@@ -1,24 +1,37 @@
 <template>
-    <section class="flex flex-col lg:flex-row px-8 py-6 border-b -mx-4">
-        <ul class="w-full px-2">
-            <li class="mb-4">
-                <menu-item
-                    id="nav_desktop_admin_menu_manage_accounts_all"
-                    title="All Users"
-                    description="List of all users"
-                    icon-class="fas fa-users"
-                    target-route-name="accountlist"
-                />
-            </li>
-            <li>
-                <menu-item
-                    id="nav_desktop_admin_menu_manage_accounts_create"
-                    title="New Account"
-                    description="Create a new user"
-                    icon-class="fas fa-user-plus"
-                    target-route-name="accountForm.create"
-                />
-            </li>
+    <section class="px-8 py-6 -mx-4">
+        <ul class="w-full px-2 space-y-4">
+            <div class="flex">
+                <li class="w-1/2">
+                    <menu-item
+                        id="nav_desktop_admin_menu_manage_accounts_all"
+                        title="All Users"
+                        description="Show every user"
+                        icon-class="fas fa-users"
+                        target-route-name="accountlist"
+                    />
+                </li>
+                <li class="w-1/2">
+                    <menu-item
+                        id="nav_desktop_admin_menu_manage_accounts_create"
+                        title="New Account"
+                        description="Create a new user"
+                        icon-class="fas fa-user-plus"
+                        target-route-name="accountForm.create"
+                    />
+                </li>
+            </div>
+            <div>
+                <li>
+                    <menu-item
+                        id="nav_desktop_admin_menu_exreg_create"
+                        title="New Exam Regulation"
+                        description="Create a new exam regulation"
+                        icon-class="fas fa-folder-plus"
+                        target-route-name="examRegForm"
+                    />
+                </li>
+            </div>
```

Comment:
It is because the lower menu entry (new exam regulation) has the full width instead of the half like the two above. If we choose to display it as 1/2 too, we need to cut its name/description.

Reply:
I know why that happens, just wanted to point it out. For example we could center the contents of the lower div :)

Reply:
Imho that would look unpolished if centred 😄  We could either squish it on the left side with shorter text or write more text so it would be obvious that it's taking the full width (which can already be seen when hovered).